tutorials/basic-topics/sorting/insertion-sort.md

```diff
@@ -11,4 +11,202 @@ keywords:
   - insertion sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Shivashish-rwt"/>
+
+## Overview
+
+Insertion sort is one of the sorting algorithms that sort the elements by placing an unsorted element in correct sorted order one at a time.
+
+First of all, let us understand how an sorted `array` (If you don't what an array is. Please refer [here](/tutorials/basic-topics/arrays.md)) looks like. Understanding an sorted array will give us the overview of the insertion sort. An sorted array is given below.
+
+Example: [1,2,3,4,5,6,7]
+
+In a sorted `array` all the elements to the left any element are smaller than that. That, means to make the `array` sorted we have to place an element in such a position, so that every element to the left are smaller than that. That is pretty much what we do in a insertion sort.
+
+## Algorithm
+
+The following are the steps that we are going to follow:
+
+1. Pick an element from the array, and store it in a variable `nums`.
```

Comment:
It would be nice, if you can take 2 numbers from the above example and walk through how it works. 

nums - Just a variable name, provide a valid examples.